SECURITY.rst

```diff
@@ -0,0 +1,129 @@
+
+
+Use of Email for Vulnerability Disclosure
+=========================================
+
+Only use the OSS security email to disclose security vulnerabilities.
+Astronomer does not accept bug reports, security implementation questions, or other security-related issues at this email address.
+If you are a customer of Astronomer, please reach out to your account team if you have any security-related questions or
+issues other than vulnerabilities, and they can assist you. Otherwise, this codebase is provided ‘as-is’ in accordance
+with its licensing structure.
+
+Scope
+=====
+
+When submitting vulnerabilities, please ensure that it is within the scope of the project. Out-of-scope vulnerability reports will be ignored.
+
+In-scope
+........
+
+* Code base with tagged releases
+* When integrated as specified in the `official Astronomer Cosmos documentation <https://astronomer.github.io/astronomer-cosmos/>`_.
+
+Out-of-scope
+............
+
+* Any other codebase, including other Astronomer products
+* Astronomer.io website
+* Dependencies used in Astronomer Cosmos
+* Astronomer Cosmos when modified or run using an unintended configuration
+* Other systems integrated with or CSP systems hosting the deployment
+* Cookie transfers between browsers
+
+For other products and repositories owned by Astronomer, please refer to their specific security policy or to
+`https://www.astronomer.io/vulnerability-disclosure/ <https://www.astronomer.io/vulnerability-disclosure/>`_ for
+vulnerabilities associated with Astronomer products.
+
+Required information and how to disclose
+====================================
+
+Please send a single, plain-text (not HTML) email for each vulnerability you report.
+At a minimum, the report must include the following:
+
+* Brief description/title of the vulnerability
+* Steps to recreate the issue
+* Contact information
+
+Upon review, we may request additional information including, but not limited to, images or a proof-of-concept video.
+
+Severity
+========
+
+The vulnerability severity rating system used internally by Astronomer is not the same as the one used by the Apache Foundation.
+Please do not provide a severity for the vulnerability when disclosing, however, providing a CWE (Common Weakness Enumeration) is recommended.
+
+Response Timeframe
+==================
+
+Astronomer aims to acknowledge and validate disclosures within 5 business days. Resolutions will be provided in a timely manner.
+
+
+Follow-up Communication
+=======================
+
+Astronomer will handle the disclosures sent to `oss_security@astronomer.io <mailto:oss_security@astronomer.io>`_ on a best-case effort,
+usually replying using the same tool.
+
+If the disclosure involves an Astronomer product or repository covered by the BugCrowd Vulnerability Disclosure Platform,
+the terms and timeframes may differ and the communication will happen via Bugcrowd.
```

Comment:
```suggestion
Astronomer handles follow-up communications to disclosures sent to [oss_security@astronomer.io](mailto:oss_security@astronomer.io) on a best-case effort, often within 3-5 business days. If the disclosure involves a product or repository that is covered by Astronomer's use of the Bugcrowd Vulnerability Disclosure Platform, please see Bugcrowd's terms of service for follow-up communication timelines. Disclosures to the Bugcrowd Vulnerability Disclosure Platform will result in communications through that platform.
```

@chaosmaw  -> is this what your comment was suggesting the changes to?

Reply:
Yes.